src/main/resources/changesets/changelog-consent-61.0.xml

```diff
@@ -0,0 +1,56 @@
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet id="61.0" author="raejohanek">
+        <validCheckSum>ANY</validCheckSum>
```

Comment:
You won't need this line. We only needed these when we did the migration from mysql to postgres. More information here https://docs.liquibase.com/concepts/basic/changeset.html